src/formatDate.js

```diff
@@ -8,7 +8,54 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separator1 = fromFormat.splice(-1, 1);
+  const separator2 = toFormat.splice(-1, 1);
```

Comment:
Using `splice` to extract the separator modifies the `fromFormat` and `toFormat` arrays. If the original arrays need to be preserved, consider using `slice` instead.